Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -222,11 +222,10 @@ bool Renderer::CalculateTargetSize(unsigned int framebuffer_width, unsigned int
 			efb_scale_denominatorX = efb_scale_denominatorY = 1;
 
 
-			int maxSize;
-			maxSize = GetMaxTextureSize();
-			if ((unsigned)maxSize < EFB_WIDTH * efb_scale_numeratorX / efb_scale_denominatorX)
+			const unsigned int max_size = static_cast<unsigned int>(GetMaxTextureSize());
```

Comment:
I think function is only used here, so just change the API to return an unsigned integer.